app/components/oidc-authorize/index.hbs

```diff
@@ -24,8 +24,11 @@
                 data-test-oidcAuthorize-scopeDescription='{{sd}}'
                 as |li|
               >
-                <li.leftIcon @disabled={{true}}>
-                  <AkIcon @iconName='chevron-right' />
+                <li.leftIcon>
+                  <AkIcon
+                    local-class='oidc-authorize-card-icon'
```

Comment:
we can use `@color='success'`

Reply:
it's not working because list item icon is given grey colour from ext. stylesheet which is taking more priority so we require !important for the colour to show.